docs/qcfractal/source/install.rst

```diff
@@ -42,6 +42,8 @@ dependencies you would like to keep up to date:
 
 *   ``pip install qcfractal``
 
+.. XXX: only one option is listed here. 
```

Comment:
Whoops, this was intended for `pip install qcfractal[extras]`. Can you rephrase to only state the singular option.